code/game/machinery/Sleeper.dm

```diff
@@ -81,9 +81,11 @@
 			dat += text("[]\t-Toxin Content %: []</FONT><BR>", (occupant.getToxLoss() < 60 ? "<font color='#487553'>" : "<font color='#b54646'>"), occupant.getToxLoss())
 			dat += text("[]\t-Burn Severity %: []</FONT><BR>", (occupant.getFireLoss() < 60 ? "<font color='#487553'>" : "<font color='#b54646'>"), occupant.getFireLoss())
 			dat += text("<HR>Knocked Out Summary %: [] ([] seconds left!)<BR>", occupant.knocked_out, round(occupant.knocked_out / 4))
-			if(occupant.reagents)
-				for(var/chemical in connected.available_chemicals)
-					dat += "[connected.available_chemicals[chemical]]: [occupant.reagents.get_reagent_amount(chemical)] units<br>"
+			for(var/chemical in connected.available_chemicals)
+				dat += "<label style='width:180px; display: inline-block'>[connected.available_chemicals[chemical]] ([occupant.reagents.get_reagent_amount(chemical)] units)</label> Inject:"
```

Comment:
```suggestion
				dat += "<label style='width:180px; display: inline-block'>[connected.available_chemicals[chemical]] ([round(occupant.reagents.get_reagent_amount(chemical), 0.01)] units)</label> Inject:"
```
So we don't get 3.333333333333-like numbers.
Feel free to use a different multiplier, I thought that `3.33` was a suitable alternative.

Reply:
3.3 might be better. I can't think of any reason to go further than that.

Reply:
Just change that to `0.1` then.

Reply:
Two significant digits is fine. it defaults to four, and the extra precision could be expected from a larger machine.